### PR TITLE
Update STS API docs with Version query param

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Update STS API docs with a Version query param

## Motivation and Context
Documentation fixes

## Regression
No

## How Has This Been Tested?
Using the new client grants and web identity APIs.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/sts/client-grants.md [hunk 1]
```diff
@@ -24,6 +24,14 @@ The OAuth 2.0 access token that is provided by the identity provider. Applicatio
 | *Length Constraints* | *Minimum length of 4. Maximum length of 2048.* |
 | *Required* | *Yes* |
 
+#### Version
+Indicates STS API version information, only supported values are only '2011-06-15'.
```
Comment (poornas) [inline docs/sts/client-grants.md]: only supported values are only -> only supported value is

File: docs/sts/web-identity.md [hunk 2]
```diff
@@ -22,6 +22,14 @@ The OAuth 2.0 access token that is provided by the web identity provider. Applic
 | *Length Constraints* | *Minimum length of 4. Maximum length of 2048.* |
 | *Required* | *Yes* |
 
+#### Version
+Indicates STS API version information, only supported values are only '2011-06-15'.
```
Comment (poornas) [inline docs/sts/web-identity.md]: only supported values are only -> only supported value is

**Review by poornas — CHANGES_REQUESTED** [review]

**Comment by harshavardhana** [comment]
ping @poornas @eco-minio PTAL

**Review by poornas — DISMISSED** [review]

File: docs/sts/client-grants.md [hunk 3]
```diff
@@ -24,6 +24,14 @@ The OAuth 2.0 access token that is provided by the identity provider. Applicatio
 | *Length Constraints* | *Minimum length of 4. Maximum length of 2048.* |
 | *Required* | *Yes* |
 
+#### Version
+Indicates STS API version information, only supported values is '2011-06-15'.
```
Comment (eco-minio) [inline docs/sts/client-grants.md]: "...only supported value is...".  
Also, do you want to include reasoning behind why this is the only supported value?

File: docs/sts/web-identity.md [hunk 4]
```diff
@@ -22,6 +22,14 @@ The OAuth 2.0 access token that is provided by the web identity provider. Applic
 | *Length Constraints* | *Minimum length of 4. Maximum length of 2048.* |
 | *Required* | *Yes* |
 
+#### Version
+Indicates STS API version information, only supported values is '2011-06-15'.
```
Comment (eco-minio) [inline docs/sts/web-identity.md]: "...only supported value..."

**Review by eco-minio — CHANGES_REQUESTED** [review]

File: docs/sts/client-grants.md (hunk 3 above)
Comment (harshavardhana) [reply docs/sts/client-grants.md]: There is no reasoning, it's an API that's the only value.

Comment (harshavardhana) [reply docs/sts/client-grants.md]: Added reasoning that its for compatibility @eco-minio PTAL

File: docs/sts/web-identity.md [hunk 5]
```diff
@@ -22,6 +22,14 @@ The OAuth 2.0 access token that is provided by the web identity provider. Applic
 | *Length Constraints* | *Minimum length of 4. Maximum length of 2048.* |
 | *Required* | *Yes* |
 
+#### Version
+Indicates STS API version information, only supported value is '2011-06-15'. This value is borrowed from AWS STS API documentation for compatibility reasons.
```
Comment (eco-minio) [inline docs/sts/web-identity.md]: "...the only supported value is..."

**Review by eco-minio — CHANGES_REQUESTED** [review]

**Review by eco-minio — APPROVED** [review]
LGTM

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Comment by harshavardhana** [comment]
ping @poornas can you approve again ?

**Review by poornas — APPROVED** [review]

**Pull request merged**